io.github.pemsley.coot.yaml

```diff
@@ -0,0 +1,346 @@
+    sources:
+      - type: git
+        url: https://github.com/rdkit/rdkit
+        tag: Release_2023_09_4
+      - type: archive
+        url: https://github.com/schrodinger/maeparser/archive/refs/tags/v1.3.1.tar.gz
+        sha256: a8d80f67d1b9be6e23b9651cb747f4a3200132e7d878a285119c86bf44568e36
+        strip-components: 0
+      - type: archive
+        url: https://github.com/schrodinger/coordgenlibs/archive/refs/tags/v3.0.2.tar.gz
+        sha256: f67697434f7fec03bca150a6d84ea0e8409f6ec49d5aab43badc5833098ff4e3
+        strip-components: 0
+      - type: archive
+        url: https://github.com/rareylab/RingDecomposerLib/archive/refs/tags/v1.1.3_rdkit.tar.gz
+        sha256: 944b5816712a48bbf88aa25d4300ce11871ddf6e971218eac08f90ed2192f715
+        strip-components: 0
+      - type: archive
+        url: https://github.com/ncbi/pubchem-align3d/archive/daefab3.tar.gz
+        sha256: 4871a08104c660890015f27d32a0147fc32df69a8962ca5cd1fe54ef6319c779
+        strip-components: 0
+      - type: archive
+        url: https://github.com/Tencent/rapidjson/archive/858451e.tar.gz
+        sha256: 765339356391422ad7f0258f8b746d8010da205762784a84980936e8b38e03ce
+        strip-components: 0
+      - type: shell
+        commands:
+          - mkdir -p External/CoordGen/{maeparser,coordgen}
+          - mkdir -p External/RingFamilies/RingDecomposerLib
+          - mkdir -p External/pubchem_shape/pubchem-align3d
+          - mkdir -p External/rapidjson-1.1.0
+          - cp -r maeparser-1.3.1/* External/CoordGen/maeparser
+          - cp -r coordgenlibs-3.0.2/* External/CoordGen/coordgen
+          - cp -r RingDecomposerLib-1.1.3_rdkit/* External/RingFamilies/RingDecomposerLib
+          - cp -r pubchem-align3d-daefab3dd0c90ca56da9d3d5e375fe4d651e6be3/* External/pubchem_shape/pubchem-align3d
+          - cp -r rapidjson-858451e5b7d1c56cf8f6d58f88cf958351837e53/* External/rapidjson-1.1.0
+
+  - name: coot
+    buildsystem: autotools
+    config-opts:
+      - --with-boost-libdir=/app/lib
+      - --with-rdkit-prefix=/app
+      - --with-enhanced-ligand-tools
+      - --with-guile
+    build-options:
+      env:
+        GLM_CFLAGS: "-I/app/include"
+        GLM_LIBS: "-L/app/lib -lglm"
+    post-install:
+      - install -Dm644 io.github.pemsley.coot.svg ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.svg
+      - install -Dm644 coot.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm644 coot.appdata.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
+    sources:
+      - type: archive
+        url: https://www2.mrc-lmb.cam.ac.uk/personal/pemsley/coot/source/releases/coot-1.1.10.tar.gz
+        sha256: 447caa7bdd6f87b738d20f8db15aa278476c308e22506004ed145e04f85e0413
+      - type: file
+        path: io.github.pemsley.coot.svg
+      - type: patch
+        path: coot.desktop.patch
+      - type: patch
+        path: coot.appdata.xml.patch
+      - type: extra-data
```

Comment:
this should NOT be extra-data. This should just be handled like any other dependency.

Extra-data is an UNRELIABLE mechanism to install thing that can't be redistributed at install time.

Reply:
> this should NOT be extra-data. This should just be handled like any other dependency.

I changed it as you advised.